pkgs/top-level/all-packages.nix

```diff
@@ -19164,6 +19170,8 @@ in
 
   kid3 = libsForQt5.callPackage ../applications/audio/kid3 { };
 
+  kio-gdrive = kdeApplications.callPackage ../applications/misc/kio-gdrive { };
```

Comment:
```suggestion
  kio-gdrive = libsForQt5.callPackage ../applications/misc/kio-gdrive { };
```

Reply:
Hello,
This package depends on some libraries from kde (e.g. libkgapi, kcalcore, kaccounts-integration). However, this package is not a part of "kde applications" set. I don't know what is the best way to provide these dependencies, but "libsForQt5.callPackage" will not work here...

Reply:
The KDE Applications libraries should be added to `mkLibsForQt5` [here](https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/all-packages.nix#L13321), it's probably just an oversight that they weren't already.